__tests__/app.test.js

```diff
@@ -0,0 +1,50 @@
+const request = require('supertest');
+const app = require('../app');
+const seed = require("../db/seeds/seed");
+const testData = require('../db/data/test-data');
+const db = require("../db/connection")
+
+beforeEach(() => {
+    return seed(testData);
+});
+
+afterAll(() => {
+    return db.end();
+});
+
+describe("GET /api/topics", () => {
+    test("200: should respond with 200 status code", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+    })
+    test("200: should respond with an array", () => {
+        return request(app)
+        .get('/api/topics')
+        .then(({body}) => {expect(body.topics).toBeInstanceOf(Array);
+        })
+    })
+    test("200: each topic should have a slug and a description", () => {
+        return request(app)
+        .get('/api/topics')
+        .then(({body}) => {
+            body.topics.forEach((topic) => {
```

Comment:
As we are using a forEach here, we should be checking the length of the array that is coming back. Could we please add another expect statement within this test that does this?